engine/function/itemparser/itemlistparser.php

```diff
@@ -1,14 +1,35 @@
 <?php 
+
 /* Returns a PHP array $id => 'name'
 	 $items = getItemList();
 	 echo $items[2160]; // Returns 'Crystal Coin'
-	*/
-function getItemList() {
-	$item_list = explode(PHP_EOL, file_get_contents('item_list.txt'));
-	$ia = array();
-	for ($i = 0; $i < count($item_list) - 1; $i++) {
-		$it = explode('@', $item_list[$i]);
-		$ia[(int)$it[0]] = ucfirst($it[1]);
+*/
+
+function getItemList() {	
+	$items = parseItems();	
+	return $items;
```

Comment:
Why do we need to assign it to `$items?`

`return parseItems()`

Reply:
Updated thanks